pkg/controller/acicontainersoperator.go

```diff
@@ -127,6 +143,51 @@ func NewAciContainersOperator(
 		0,
 		cache.Indexers{},
 	)
+	node_informer := cache.NewSharedIndexInformer(
+		&cache.ListWatch{
+			ListFunc: func(options metav1.ListOptions) (runtime.Object, error) {
+				return k8sclient.CoreV1().Nodes().List(options)
+			},
+			WatchFunc: func(options metav1.ListOptions) (watch.Interface, error) {
+				return k8sclient.CoreV1().Nodes().Watch(options)
+			},
+		},
+		&v1.Node{},
+		0,
+		cache.Indexers{},
+	)
+	var routesClient routesClientset.Interface
+	var route_informer cache.SharedIndexInformer
+	flavor := os.Getenv("FLAVOR")
```

Comment:
This string is too generic at the env level (someone can accidentally override it). Suggest calling it "ACC_PROVISION_FLAVOR". Also, what happens if its not present?

Reply:
I will change it

Reply:
if it is not there it returns the nil string